pkg/platform/kube/platform.go

```diff
@@ -290,6 +290,14 @@ func (p *Platform) CreateFunction(ctx context.Context, createFunctionOptions *pl
 			}
 		}
 
+		if functionStatus.State == functionconfig.FunctionStateUnhealthy {
+			createFunctionOptions.Logger.WarnWithCtx(ctx, "Function deployment failed, setting state to unhealthy. The issue might be transient or require manual redeployment",
+				"err", creationError)
+		} else {
+			createFunctionOptions.Logger.WarnWithCtx(ctx, "Function creation failed, setting state to error",
+				"err", creationError)
+		}
```

Comment:
remove the logs from lines 265-269.
See how the `briefErrorMessage` is logged as `debug` there - do the same here